### PR TITLE
release, TOC: add release notes for 2.1.11

### PR DESCRIPTION
**Pull request by dcalvin**
Added release notes for 2.1.11
PTAL @CaitinChen @ericsyh

### CODE REVIEW

File: releases/2.1.11.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+TiDB 版本：2.1.11
+
+TiDB Ansible 版本：2.1.11
+
+## TiDB
+
+- 修复 delete 多表 join 的结果时使用错误 schema 的问题 [#10595](https://github.com/pingcap/tidb/pull/10595)
+- 修复 `CONVERT()` 函数返回错误的字段类型的问题 [#10263](https://github.com/pingcap/tidb/pull/10263)
+- 更新统计信息时合并不重叠的反馈信息[#10569](https://github.com/pingcap/tidb/pull/10569)
```
Comment (CaitinChen) [inline releases/2.1.11.md]:
```suggestion
- 更新统计信息时合并不重叠的反馈信息 [#10569](https://github.com/pingcap/tidb/pull/10569)
```

File: releases/2.1.11.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- 收集统计信息的时候，忽略 `Virtual Column`，避免异常报错 [#10628](https://github.com/pingcap/tidb/pull/10628)
+- 支持 `SHOW OPEN TABLES` 语句 [#10374](https://github.com/pingcap/tidb/pull/10374)
+- 修复某些情况下导致的 goroutine 泄露问题 [#10656](https://github.com/pingcap/tidb/pull/10656)
+- 修复某些情况下设置 `tidb_snapshot` 变量时间格式解析出错的问题 [#10637](https://github.com/pingcap/tidb/pull/10637)
+
+
+## PD
+
+- 修复因为 `balance-region` 可能会导致热点 Region 没有机会调度的问题
+[#1551](https://github.com/pingcap/pd/pull/1551)
```
Comment (CaitinChen) [inline releases/2.1.11.md]: Please remove it to the line above.

File: releases/2.1.11.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- 将热点相关调度的优先级改为高优先级
+[#1551](https://github.com/pingcap/pd/pull/1551)
```
Comment (CaitinChen) [inline releases/2.1.11.md]: ditto

File: releases/2.1.11.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- 新增配置项 `hot-region-schedule-limit` 控制同时进行热点调度任务的数量及新增 `hot-region-cache-hits-threshold` 控制判断是否为热点 Region
+[#1551](https://github.com/pingcap/pd/pull/1551)
```
Comment (CaitinChen) [inline releases/2.1.11.md]: ditto

File: releases/2.1.11.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## TiKV
+
+- 修复在仅有一个 leader，learner 时，learner 读到空 index 的问题
+[#4751](https://github.com/tikv/tikv/pull/4751)
```
Comment (CaitinChen) [inline releases/2.1.11.md]: ditto

**Comment by ericsyh** [comment]
LGTM to Tools

File: releases/2.1.11.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- 将 `ScanLock` 和 `ResolveLock` 放在高优先级线程池中处理，减少对普通优先级命令的影响 [#4791](https://github.com/tikv/tikv/pull/4791)
+
+## Tools
+
+- TiDB Binlog
+  - 新增 GC 删数据限速功能，避免因为删除数据导致 QPS 降低的问题 [#620](https://github.com/pingcap/tidb-binlog/pull/620)
```
Comment (CaitinChen) [inline releases/2.1.11.md]:
```suggestion
    - 新增 GC 删数据限速功能，避免因为删除数据导致 QPS 降低的问题 [#620](https://github.com/pingcap/tidb-binlog/pull/620)
```

File: releases/2.1.11.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- 修复 delete 多表 join 的结果时使用错误 schema 的问题 [#10595](https://github.com/pingcap/tidb/pull/10595)
+- 修复 `CONVERT()` 函数返回错误的字段类型的问题 [#10263](https://github.com/pingcap/tidb/pull/10263)
+- 更新统计信息时合并不重叠的反馈信息[#10569](https://github.com/pingcap/tidb/pull/10569)
+- 修复  `unix_timestamp()-unix_timestamp(now())` 计算错误的问题 [#10491](https://github.com/pingcap/tidb/pull/10491)
+- 修复 `period_diff` 与 MySQL 8.0 不兼容的问题 [#10501](https://github.com/pingcap/tidb/pull/10501)
+- 收集统计信息的时候，忽略 `Virtual Column`，避免异常报错 [#10628](https://github.com/pingcap/tidb/pull/10628)
+- 支持 `SHOW OPEN TABLES` 语句 [#10374](https://github.com/pingcap/tidb/pull/10374)
+- 修复某些情况下导致的 goroutine 泄露问题 [#10656](https://github.com/pingcap/tidb/pull/10656)
+- 修复某些情况下设置 `tidb_snapshot` 变量时间格式解析出错的问题 [#10637](https://github.com/pingcap/tidb/pull/10637)
+
```
Comment (dcalvin) [inline releases/2.1.11.md]:
```suggestion
```

File: releases/2.1.11.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+## PD
+
+- 修复因为 `balance-region` 可能会导致热点 Region 没有机会调度的问题
+[#1551](https://github.com/pingcap/pd/pull/1551)
+- 将热点相关调度的优先级改为高优先级
```
Comment (dcalvin) [inline releases/2.1.11.md]:
```suggestion
- 将热点相关调度的优先级改为高优先级 
```

File: releases/2.1.11.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+[#1551](https://github.com/pingcap/pd/pull/1551)
+- 新增配置项 `hot-region-schedule-limit` 控制同时进行热点调度任务的数量及新增 `hot-region-cache-hits-threshold` 控制判断是否为热点 Region
```
Comment (dcalvin) [inline releases/2.1.11.md]:
```suggestion
- 新增配置项 `hot-region-schedule-limit` 控制同时进行热点调度任务的数量及新增 `hot-region-cache-hits-threshold` 控制判断是否为热点 Region 
```

File: releases/2.1.11.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+[#1551](https://github.com/pingcap/pd/pull/1551)
+
+## TiKV
+
+- 修复在仅有一个 leader，learner 时，learner 读到空 index 的问题
```
Comment (dcalvin) [inline releases/2.1.11.md]:
```suggestion
- 修复在仅有一个 leader，learner 时，learner 读到空 index 的问题 
```

**Pull request merged**